### PR TITLE
[Bug Fix] Handle interfaces with no inputField or fields.

### LINKED ISSUE #35: Handle attributes with no inputFields or fields
We [have an enum](https://github.com/artsy/metaphysics/blob/b2f51d0d57bbe064b623bfda94fa3d2463d8ed2f/schema/show.js#L125-L147) with no fields, or inputFields [in the schema.json](https://github.com/alloy/relational-theory/blob/204609edfc99ee1fc55e6d9a60913ce24bcc65bd/data/schema.json#L7500)

```json
        {
          "kind": "UNION",
          "name": "PartnerTypes",
          "description": null,
          "fields": null,
          "inputFields": null,
          "interfaces": null,
          "enumValues": null,
          "possibleTypes": [
            {
              "kind": "OBJECT",
              "name": "Partner",
              "ofType": null
            },
            {
              "kind": "OBJECT",
              "name": "ExternalPartner",
              "ofType": null
            }
          ]
        },
```

This currently crashes the process. Will send a PR.

### PR DESCRIPTION
**Pull request by orta**
Fixes #35

### CODE REVIEW

File: util/interface.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ const typeToInterface = (type, ignoredTypes) => {
 
   let isInput = type.kind === 'INPUT_OBJECT';
   let f = isInput ? type.inputFields : type.fields;
+  f = f ? f : []
```
Comment (brettjurgens) [inline util/interface.js]: minor nitpick, would probably do `f || []`

**Comment by brettjurgens** [comment]
nice 👍

ideally we'd add test coverage as well, but I don't think it's necessary

**Comment by orta** [comment]
I went with the same conclusion re:tests, it'd be a lot of JSON for a single test - updated

**Comment by brettjurgens** [comment]
agreed, merging 👍

**Pull request merged**